public_app/components/Details.js

```diff
@@ -0,0 +1,36 @@
+import React    from 'react';
+
+class Details extends React.Component {
+    render (){
+        return (
+            <div className="detail-container">
+                <div className='detail-img'>
+                    <img src={this.props.src} alt={this.props.title} />
```

Comment:
can destructure to `const { src, title, released, genre, actors } = this.props;` so you can access the values directly with for example `{title}` instead of `this.props.title`